config/locales/en.yml

```diff
@@ -225,6 +225,9 @@ en:
       transparency:
         one: Transparency data
         other: Transparency data
+      world_location_news_article:
```

Comment:
so this is currently used when rendering the history notice (https://github.com/alphagov/government-frontend/blob/master/app/views/shared/_history_notice.html.erb#L3).

I notice that 'news_article' is already translated - so do we want to fall back to that for our translation? Otherwise this probably needs to be translated